Automattic/jetpack
# Social: Pull dynamic values from DB in pricing table

This change replaces the hardcoded values on the Social admin page, and pull the current prices based on the users locale.

## Proposed changes:

* Added new endpoint to fetch the product info
* Created a hook that calls the endpoint
* Used the hook to get the data. Spinners are present while loading

### Other information:


## Jetpack product discussion



1204272127368434-as-1204008786549223

## Does this pull request change what data or activity we track or use?



No.

## Testing instructions:






* Go to the Social admin page
* In `projects/plugins/social/src/js/components/admin-page/index.jsx` change the logic so the `PricingPage` is shown, or create a new site.
* Test that the prices are pulled in correctly. Should be the same as on https://jetpack.com/social/
